zuulcilint/__main__.py

```diff
@@ -101,39 +101,70 @@ def get_all_jobs(zuul_yaml_files: list[pathlib.Path]) -> list[list[str]]:
     return all_jobs
 
 
-def print_warnings(bad_yml_files: list[str], duplicated_jobs: set[str]) -> None:
+def print_warnings(bad_yml_files: list[str], duplicated_jobs: set[str], severity="warning") -> None:
     """Print warnings."""
```

Comment:
missing str type for severity

```suggestion
def print_warnings(bad_yml_files: list[str], duplicated_jobs: set[str], severity: str = "warning") -> None:
```